modules/ROOT/pages/_partials/link-anypoint-platform-accounts.adoc

```diff
@@ -0,0 +1,94 @@
+// tag::profileLinkingContextForUsers[]
+[[profile-linking-context-users]]
+= Linking Your Anypoint Platform Profiles
+
+When you log in to an Anypoint Platform using an email address that is associated with more than one profile in the same organization, you are prompted to link your profiles together. Your options for linking profiles vary based on whether the profiles use the Anypoint identity provider (IdP) or an external IdP that uses single sign-on (SSO). Anypoint Platform supports:
```

Comment:
Let me think hard about this : "When you log in to an Anypoint Platform using an email address that is associated with more than one profile in the same organization" 

The user is not actually using an email address to login. The users are logging in from an Idp (i.e. OpenId Idp at this release). The end users may not be immediately aware of which emails they have in their Idp account. Let me take an example. If the Idp requires username (not email) to sign in, the end users type username/password, get redirected to our linking page and see the matched accounts. In this process the email is not displayed. 

What's actually happening is that: (this is for the context; I'm not sure how we put it in here) 
1. the user signs into the IDP 
2. IDP sends to Anypoint a profile (username, email, ... stored in IDP)
3. we use that email to find potential matches. 

So I would say something like "When you log into Anypoint via an identity provider for the first time, we'll find potential user accounts matched by your email address in the Idp profile. You'll be prompted to choose to link your current idp profile to an existing Anypoint account... "